staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -299,13 +307,15 @@ type UIDMap map[ResourceLocation]types.UID
 // WaitOptions is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
 // command, making the logic itself easy to unit test with our existing mocks.
 type WaitOptions struct {
-	ResourceFinder genericclioptions.ResourceFinder
+	ResourceFinder        genericclioptions.ResourceFinder
+	CreationCheckerFinder genericclioptions.ResourceFinder
```

Comment:
I'm trying to imagine a scenario where this would need to be different than `ResourceFinder`.

Can `ResourceFinder` be used to wait for creation as well?  This would eliminate the need to create another finder.

Reply:
I mean can you do this without `CreationCheckerFinder` and just use `ResourceFinder` in both places?

Reply:
Thanks for review @brianpursley. You are right, `CreationCheckerFinder` seems to be unnecessary and removed.